keystore/encrypt_test.go

```diff
@@ -8,7 +8,8 @@ import (
 	"reflect"
 	"testing"
 
-	"github.com/ChainSafe/gossamer/crypto"
+	"github.com/ChainSafe/gossamer/crypto/ed25519"
+	"github.com/ChainSafe/gossamer/crypto/sr25519"
```

Comment:
Should we include secp in keystore encrypt test?

Reply:
done